### ISSUE TITLE
xmlFix fails with Exhange 2007 wsdl

### ISSUE THREAD

**Issue by eino-makitalo** [nmarus/node-ews#31]
Unexpected close tag
Line: 1023
Column: 19
Char: >

**Comment by eino-makitalo** [comment]
Last rows of Services.wsdl are:
````xml
<!--  SetUserOofSettings  -->
<wsdl:operation name="SetUserOofSettings">
<soap:operation soapAction="http://schemas.microsoft.com/exchange/services/2006/messages/SetUserOofSettings"/>
<wsdl:input>
<soap:body parts="SetUserOofSettingsRequest" use="literal"/>
</wsdl:input>
<wsdl:output>
<soap:body parts="SetUserOofSettingsResult" use="literal"/>
<soap:header message="tns:SetUserOofSettingsSoapOut" part="ServerVersion" use="literal"/>
</wsdl:output>
</wsdl:operation>
</wsdl:binding>
</wsdl:definitions>
````

**Comment by eino-makitalo** [comment]
I changed           wsdlLines.splice(-2,2);  to           wsdlLines.splice(-1,1); and now it works

**Comment by nmarus** [comment]
Resolved in  v2.1.7 and v3.0.0
